includes/razorpay-webhook.php

```diff
@@ -100,7 +100,9 @@ public function process()
                 return;
             }
             if (isset($_SERVER['HTTP_X_RAZORPAY_SIGNATURE']) === true) {
-                $razorpayWebhookSecret = $this->razorpay->getSetting('webhook_secret');
+                
+                $razorpayWebhookSecret = (empty($this->razorpay->getSetting('webhook_secret')) === false) ? $this->razorpay->getSetting('webhook_secret') : get_option('webhook_secret');
```

Comment:
@yashgit891 do you have idea why there is another webhook secret rzp_webhook_secret flag

Reply:
@abdulwahidsharief it was present for few of the older merchants before the flag was renamed to webhook_secret.